docs/data/Allas/introduction.md

```diff
@@ -28,22 +28,22 @@ The stored objects can be of any data type, such as images or compressed data fi
  * Files larger than 5 GB must be divided into smaller segments (normally this is done automatically during upload), see [Files larger than 5 GB](./using_allas/common_use_cases.md#files-larger-than-5-gb).
 
 
-## Different flavors for using Allas
```

Comment:
Or ways? clients is already quite specific

Reply:
reworded.